### PR TITLE
Added lua .insert()

### PR DESCRIPTION
**Pull request by pascualcam**
### Description

Added Lua .insert() function metadata.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by SSwiniarski** [comment]
@pascualcam There are a couple of things you need to correct in this entry before it gets reviewed.

1) This is in the wrong location. The file path for the .md file as specified by the issue should be `docs/content/lua/concepts/tables/terms/insert/insert.md`
2) You are missing a metadata section for this entry. Please review the instructions for entering metadata from the style guide [here](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#metadata).

Tag me when these issues are addressed and I'll conduct a fist review.

**Comment by pascualcam** [comment]
@SSwiniarski I added the metadata as shown in the sample and added the .md file to the appropriate folder. Cannot delete the previous entry that went into the /documentation directory.

**Comment by SSwiniarski** [comment]
@pascualcam the file needs to be located at `content/lua/concepts/tables/terms/insert/insert.md` (note that it is in a folder named insert under `/terms`)

Also you need to remove the extra blank lines in your metadata.

Why can you not delete the file in the wrong place? How are you editing this PR?

**Comment by pascualcam** [comment]
@SSwiniarski I believe I have fixed both of your comments.

File: content/lua/concepts/tables/terms/insert.md
Comment (SSwiniarski) [inline content/lua/concepts/tables/terms/insert.md:1]: This file needs to be moved to `content/lua/concepts/tables/terms/insert/insert.md`

File: documentation/insert.md
Comment (SSwiniarski) [inline documentation/insert.md:1]: This file needs to be deleted.

File: content/lua/concepts/tables/terms/insert.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Tables'
+  - 'Functions'
+CatalogContent:
+  - 'learn-lua'
+  - 'paths/computer-science'
+---
+
+The **`.insert()`** method inserts a value in a table.
```
Comment (SSwiniarski) [inline content/lua/concepts/tables/terms/insert.md:14]:
```suggestion
The **`.insert()`** method inserts a value in a table at a specified index.
```

File: content/lua/concepts/tables/terms/insert.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+## Syntax
+
+The `value` can be inserted at a specific `index`.  
+
```
Comment (SSwiniarski) [inline content/lua/concepts/tables/terms/insert.md:19]:
```suggestion
```
Delete this line.

File: content/lua/concepts/tables/terms/insert.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+```lua
+table.insert(table, index, value)
+```
+
+If `index` is not provided, `value` is inserted at the end of `table`. It returns `table`. Stacks, queues, and double queues can be implemented in this way.
```
Comment (SSwiniarski) [inline content/lua/concepts/tables/terms/insert.md:24]:
```suggestion
The `value` is inserted at `index`. If `index` is not provided, `value` is inserted at the end of `table`. It returns `table`. Stacks, queues, and double queues can be implemented in this way.    
```

File: content/lua/concepts/tables/terms/insert.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+The `value` can be inserted at a specific `index`.  
+
+```lua
+table.insert(table, index, value)
+```
+
+If `index` is not provided, `value` is inserted at the end of `table`. It returns `table`. Stacks, queues, and double queues can be implemented in this way.    
+
+## Codebyte Example
```
Comment (SSwiniarski) [inline content/lua/concepts/tables/terms/insert.md:26]:
```suggestion
## Example
```
We don't have codebytes implemented for lua.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@pascualcam I have a few review comments. You still need to move the file and delete the extra one. Are you having trouble doing this?

File: content/lua/concepts/tables/terms/insert.md
Comment (pascualcam) [reply content/lua/concepts/tables/terms/insert.md:1]: Done

File: documentation/insert.md
Comment (pascualcam) [reply documentation/insert.md:1]: Removed

File: content/lua/concepts/tables/terms/insert.md (hunk 1 above)
Comment (pascualcam) [reply content/lua/concepts/tables/terms/insert.md:14]: Implemented suggested updates

File: content/lua/concepts/tables/terms/insert.md (hunk 2 above)
Comment (pascualcam) [reply content/lua/concepts/tables/terms/insert.md:19]: Implemented suggested updates

File: content/lua/concepts/tables/terms/insert.md (hunk 3 above)
Comment (pascualcam) [reply content/lua/concepts/tables/terms/insert.md:24]: Implemented suggested updates

File: content/lua/concepts/tables/terms/insert.md (hunk 4 above)
Comment (pascualcam) [reply content/lua/concepts/tables/terms/insert.md:26]: Implemented suggested updates

**Comment by pascualcam** [comment]
Let me know if you have further comments. Thanks for your feedback and patience while I become more acquainted with git.

**Review by SSwiniarski — APPROVED** [review]
@pascualcam I deleted the extra files since you seemed to have difficulty with it. I also made a few other small edits. Now it looks good for a second review.

File: content/lua/concepts/tables/terms/insert/insert.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+CatalogContent:
+  - 'learn-lua'
+  - 'paths/computer-science'
+---
+
+The **`.insert()`** method inserts a value in a table at a specified index.
+
+## Syntax
+
+```lua
```
Comment (caupolicandiaz) [inline content/lua/concepts/tables/terms/insert/insert.md:18]:
```suggestion
```pseudo
```

File: content/lua/concepts/tables/terms/insert/insert.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+table.insert(table, index, value)
```
Comment (caupolicandiaz) [inline content/lua/concepts/tables/terms/insert/insert.md:19]:
```suggestion
table.insert(sometable, index, value)
```

File: content/lua/concepts/tables/terms/insert/insert.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+```
+
+The `value` is inserted at `index`. If `index` is not provided, `value` is inserted at the end of `table`. It returns `table`. Stacks, queues, and double queues can be implemented in this way.
```
Comment (caupolicandiaz) [inline content/lua/concepts/tables/terms/insert/insert.md:22]:
```suggestion
The `value` is inserted at `index`. If `index` is not provided, `value` is inserted at the end of `sometable`. The function returns the modified, original table.

 Stacks, queues, and double queues can be implemented in this way.
```

File: content/lua/concepts/tables/terms/insert/insert.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+apple_products = {"Macbook", "Watch", "iPad"}
+table.insert(apple_products, "iPhone")
+for key, value in  pairs(apple_products) do
+	print(key, value)
+end
+```
+
+The output would be:
+
+```
```
Comment (caupolicandiaz) [inline content/lua/concepts/tables/terms/insert/insert.md:38]:
```suggestion
```shell
```

File: content/lua/concepts/tables/terms/insert/insert.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+
+```lua
+apple_products = {"Macbook", "Watch", "iPad"}
+table.insert(apple_products, "iPhone")
+for key, value in  pairs(apple_products) do
+	print(key, value)
+end
+```
+
+The output would be:
```
Comment (caupolicandiaz) [inline content/lua/concepts/tables/terms/insert/insert.md:36]:
```suggestion
The code above results in the following output:
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@pascualcam I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by pascualcam** [comment]
@caupolicandiaz I have implemented your suggested edits

**Review by caupolicandiaz — APPROVED** [review]
@pascualcam @SSwiniarski Looks good to merge

**Pull request merged**